components/system/components/CodeBlock.js

```diff
@@ -1,13 +1,15 @@
-import * as React from 'react';
-import * as Constants from '~/common/constants';
+import * as React from "react";
+import * as Constants from "~/common/constants";
 
-import { css } from '@emotion/react';
+import { css } from "@emotion/react";
```

Comment:
LGTM! We will want to make sure your prettier config is updated at some point so our formatting stays the same.